frontend/src/tests/pages/MenuItemReview/MenuItemReviewCreatePage.test.js

```diff
@@ -0,0 +1,126 @@
+jest.mock('react-toastify', () => {
+    const originalModule = jest.requireActual('react-toastify');
+    return {
+        __esModule: true,
+        ...originalModule,
+        toast: (x) => mockToast(x)
+    };
+});
+
+const mockNavigate = jest.fn();
+jest.mock('react-router-dom', () => {
+    const originalModule = jest.requireActual('react-router-dom');
+    return {
+        __esModule: true,
+        ...originalModule,
+        Navigate: (x) => { mockNavigate(x); return null; }
+    };
+});
+
+
+describe("MenuItemReviewCreatePage tests", () => {
+
+    const axiosMock = new AxiosMockAdapter(axios);
+
+    beforeEach(() => {
+        jest.clearAllMocks();
+        axiosMock.reset();
+        axiosMock.resetHistory();
+        axiosMock.onGet("/api/currentUser").reply(200, apiCurrentUserFixtures.userOnly);
+        axiosMock.onGet("/api/systemInfo").reply(200, systemInfoFixtures.showingNeither);
+    });
+
+    const queryClient = new QueryClient();
+    test("renders without crashing", () => {
+        render(
+            <QueryClientProvider client={queryClient}>
+                <MemoryRouter>
+                    <MenuItemReviewCreatePage />
+                </MemoryRouter>
+            </QueryClientProvider>
+        );
+    });
+
+    test("on submit, makes request to backend, and redirects to /menuitemreviews", async () => {
+
+        const queryClient = new QueryClient();
+        const review = {
+            id: 3,
+            itemId: 3,
+            reviewerEmail: "testemail@ucsb.edu",
+            stars: 3,
+            dateReviewed: "2022-07-04T12:00",
+            comments: "Barely Passable"
+        };
+
+        axiosMock.onPost("/api/menuitemreview/post").reply(202, review);
+
+        render(
+            <QueryClientProvider client={queryClient}>
+                <MemoryRouter>
+                    <MenuItemReviewCreatePage />
+                </MemoryRouter>
+            </QueryClientProvider>
+        )
+
+        await waitFor(() => {
+            expect(screen.getByLabelText("Menu Item ID")).toBeInTheDocument();
+        });
+
+        const itemIdInput = screen.getByLabelText("Menu Item ID");
+        expect(itemIdInput).toBeInTheDocument();
+
+        const reviewerEmailInput = screen.getByLabelText("Email");
+        expect(reviewerEmailInput).toBeInTheDocument();
+
+        const starsInput = screen.getByLabelText("Stars");
+        expect(starsInput).toBeInTheDocument();
+
+        const dateReviewedInput = screen.getByLabelText("Date (iso format)");
+        expect(dateReviewedInput).toBeInTheDocument();
+
+        const commentsInput = screen.getByLabelText("Comments");
+        expect(commentsInput).toBeInTheDocument();
+
+        const createButton = screen.getByText("Create");
+        expect(createButton).toBeInTheDocument();
+
+        fireEvent.change(itemIdInput, { target: { value: 3 } })
+        fireEvent.change(reviewerEmailInput, { target: { value: 'testemail@ucsb.edu' } })
```

Comment:
In your MenuItemCreateForm.js file you should check if the email input string has the @ symbol. I asked the TA and they said that they're tests will be checking for it.

Reply:
I added checks for stars being less than 1 and the @ symbol for the email